### DISCUSSION TITLE
Any plan to intergrate Staticman as a comment function?

### DISCUSSION THREAD

**Discussion by ewonchang** [adityatelange/hugo-PaperMod#176] [Ideas]
Personally I like the idea of Staticman as a comment system for static websites. I know a few Hugo themes support Staticman out of the box, such as Beautiful Hugo. 

I guess I can hack around to add code snippets but I'm not technically confident enough to take the risk.

Any chance PaperMod will support Staticman? Thanks.

**Comment by adityatelange** [comment]
We don't support it built in but you can add it on you on by pasting the form snippet in comments.html
https://adityatelange.github.io/hugo-PaperMod/posts/papermod/papermod-features/#comments

and custom head/footer assets by using 
https://adityatelange.github.io/hugo-PaperMod/posts/papermod/papermod-faq/#custom-head--footer
